# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/vlc/hotfix-upnp22.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- a/modules/services_discovery/upnp.hpp
+++ b/modules/services_discovery/upnp.hpp
@@ -35,7 +35,7 @@
 
 #include <vlc_common.h>
 
-#if UPNP_VERSION < 10800
+#if UPNP_VERSION < 10800 || UPNP_VERSION >= 220000
 typedef void* UpnpEventPtr;
 #else
 typedef const void* UpnpEventPtr;
